### PR TITLE
Fix release script and prepare for release

### PR DESCRIPTION
**Pull request by YJDoc2**
This fixes the release script, and updates crate versions in preparation for release

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -72,43 +72,31 @@ jobs:
...
-          echo "::set-output name=innerdir::youki-${VERSION}"
-      - name: Create Release Draft
-        id: create_release
-        uses: actions/create-release@v1
-        env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-        with:
-          tag_name: ${{ github.ref }}
-          release_name: ${{ steps.info.outputs.version }} Release
-          draft: true
```
Comment (YJDoc2) [inline .github/workflows/release.yml:86]: This has to be removed, as the action is deprecated

File: README.md [hunk 2]
```diff
@@ -151,7 +151,8 @@ $ sudo apt-get install   \
       libdbus-glib-1-dev \
       build-essential    \
       libelf-dev \
-      libseccomp-dev
+      libseccomp-dev \
+      libclang-dev
```
Comment (YJDoc2) [inline README.md:155]: Had to add this, otherwise it wouldn't compile on my system. @utam0k can you check?

Comment (utam0k) [reply README.md:155]: sorry but I don't have my laptop now. I wonder why the CI passde without it.

**Comment by YJDoc2** [comment]
Hey @utam0k , actually this needs a few improvements, I'll update it as soon as I can

**Comment by YJDoc2** [comment]
@utam0k Please take a look
This should fix the script. Now when we push a new tag, it will run tests, create a release draft and publish the packages on crates.io (provided the secret is set to the crates io key correctly)
Also, I request you that for this PR, please do a squash and merge instead of normal merge, as there are a lot of temp/testing commits I had to do

**Comment by utam0k** [comment]
I set my token into this repo as the secret.
publish the packages on crates.io (provided the secret is set to the crates io key correctly)

**Review by utam0k — APPROVED** [review]

**Pull request merged**